### PR TITLE
Final

### PR DESCRIPTION
**Pull request by Oluwadamilareolusakin**
This PR finishes up the calculator functionality by hooking up the handleClick method to each button and finalising the logic in calculate and operate

### CODE REVIEW

**Comment by Oluwadamilareolusakin** [comment]
Apologies, I accidentally merged this into development, however, if you can review and let me know if there are necessary changes, I will appreciate that!

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
-          <Button value = {c} className = {d ? 'button' : 'button orange-btn'} />
-          {d && <Button value = {d} className = 'button orange-btn'/>}
-        </div>)
+      return (
+        <div key={groups.indexOf(group)}>
+          <Button value={a} wide={a === '0' ? true : false} backgroundColor = "#EEE" onClick={handleClick}/>
+          <Button value={b} backgroundColor = "#EEE" onClick={handleClick}/>
+          <Button value={c} backgroundColor = {d && "#EEE"}  onClick={handleClick}/>
+          {d && <Button value={d} onClick={handleClick}/>}
+        </div>
```
Comment (shloch) [inline src/components/ButtonPanel.js]: it's tricky/confusing sometimes to pass a prop called `onClick` lol, sometimes we confuse it with javascript's `"onClick"`

That's why this project proposes the name "`clickHandler`" [ [SCREENSHOT](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/passToButton.png) ]

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  }
+
+  if(operation === '-'){
+    return numberOne.minus(numberTwo);
+  }
+
+  if(operation === '/'){
+    return numberOne.div(numberTwo);
+  }
+
```
Comment (shloch) [inline src/logic/operate.js]: You have not handled division by ZERO, so it generates an error
Please handle this 
Screen Shot 2019-12-27 at 1 19 23 PM

Comment (shloch) [reply src/logic/operate.js]: You can display 'INFINITY' on screen or some other error management strategy you can think of

**Pull request merged**